datastore-api/tests/test_kgs.py

```diff
@@ -11,10 +13,10 @@ def test_get_all_kgs(self, client: TestClient, conceptnet_kg: Datastore):
         
         # When:
         response = client.get(url)
-        
+
         # Then:
         assert response.status_code == expected_code
-        assert len(response.json()) == 1
+        assert len(response.json()) >= 1
         assert sorted(response.json()[0]['fields'], key=lambda x: x['name']) == sorted(conceptnet_kg.dict()['fields'], key=lambda x: x['name'])
```

Comment:
This is different from the master branch https://github.com/UKP-SQuARE/square-core/blob/d659549f85ca2f769353c6abea9d383ea8261d66/datastore-api/tests/test_kgs.py#L24-L41 Please pull it first